### PR TITLE
Fix(39): run dev sheet error

### PR DESCRIPTION
**Pull request by boychina**
1. 修复pnpm run dev:sheet报错问题；
2. 修复部分eslint问题；

### CODE REVIEW

File: packages/base-docs/src/View/Render/Views/DocsView.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export class DocsView extends BaseView {
 
     constructor(
         @Inject(CommandManager) private readonly _commandManager: CommandManager,
-        @ICurrentUniverService private readonly _currentUniverService: ICurrentUniverService,
+        @IDCurrentUniverService private readonly _currentUniverService: ICurrentUniverService,
```
Comment (wzhudev) [inline packages/base-docs/src/View/Render/Views/DocsView.ts:21]: Please do not rename this service

File: packages/core/src/Sheets/Domain/Range.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import { ICurrentUniverService } from 'src/Service/Current.service';
+import { Inject } from '@wendellhu/redi';
+import { IDCurrentUniverService, ICurrentUniverService } from '../../Service/Current.service';
 
 // TODO@wzhudev: here we still have some
 
-import { Inject } from '@wendellhu/redi';
+// eslint-disable-next-line import/no-cycle
```
Comment (wzhudev) [inline packages/core/src/Sheets/Domain/Range.ts:6]: Please do not silent this warning. We intent to fix this later.

File: packages/core/src/Sheets/Domain/Range.ts [hunk 3]
```diff
@@ -347,9 +347,12 @@ export class Range {
      */
     getFontColors(): Array<Array<Nullable<string>>> {
         const styles = this._currentUniverService.getCurrentUniverSheetInstance().getWorkBook().getStyles();
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```
Comment (wzhudev) [inline packages/core/src/Sheets/Domain/Range.ts:350]: Please do not add these.

**Review by wzhudev — CHANGES_REQUESTED** [review]
Thanks for openning this. Though some problems should be fixed before we merging this.

File: packages/core/src/Sheets/Domain/Range.ts (hunk 2 above)
Comment (boychina) [reply packages/core/src/Sheets/Domain/Range.ts]: fixed

File: packages/base-docs/src/View/Render/Views/DocsView.ts (hunk 1 above)
Comment (boychina) [reply packages/base-docs/src/View/Render/Views/DocsView.ts]: fixed

**Comment by wzhudev** [comment]
Another name such as "style(*): fix lint errors" would be more appropriate

**Comment by jikkai** [comment]
Already solved this issue in PR #128

**Pull request closed without merging**